index.html

```diff
@@ -78,33 +78,41 @@ <h2>
       The API is designed to support the following goals:
     </p>
     <ul>
-      <li>Keep the act of requesting separate from the specific [=digital
+      <li>Keep the acts of requesting and issuance separate from the specific [=digital
       credential/protocol=], thereby enabling the extensibility of the
       [=digital credential/protocol=] and credential formats.
       </li>
-      <li>Require [=digital credential/requests=] to be unencrypted, enabling
+      <li>Require [=digital credential/presentation requests=] and 
+      [=digital credential/issuance requests=] to be unencrypted, enabling
       user-agent inspection for risk analysis.
       </li>
-      <li>Assume [=digital credential/presentation response|responses=] opacity
-      (encrypted responses), enabling verifiers and holders to control where
+      <li>Assume opaque (i.e., encrypted) [=digital credential/presentation responses=]
+      and [=digital credential/issuance responses=],
+      enabling issuers, verifiers and holders to control where
       potentially sensitive personally identifiable information is exposed.
       </li>
       <li>Require [=transient activation=] to perform [=digital
-      credential/requests=], ensuring that sites cannot silently query for
-      digital credentials nor communicate with wallet providers without the
-      user's active participation and confirmation of each action.
+      credential/presentation requests=] or [=digital
+      credential/issuance requests=], ensuring that sites cannot silently query for
+      nor offer digital credentials, nor communicate with wallet providers, without
+      the user's active participation and confirmation of each action.
       </li>
       <li>Enable platform-provided credential selection UX when multiple wallet
       applications have credentials that match a [=digital
-      credential/requests=].
+      credential/presentation request=] or [=digital
+      credential/issuance request=].
```

Comment:
```suggestion
      credential/presentation request=].
```
A separate bullet for issuance was added below.